app/controllers/state_file/questions/az_public_school_contributions_controller.rb

```diff
@@ -24,20 +26,22 @@ def build_contribution
       end
 
       def create
-        @az322_contribution = current_intake.az322_contributions.build(az322_contribution_params)
-        @az322_contributions = current_intake.az322_contributions
-        if @az322_contribution.made_contribution_no?
-          return redirect_to next_path
-        end
+        @az322_contribution = current_intake.az322_contributions.build
+        @az322_contribution.assign_attributes(az322_contribution_params)
 
-        if current_intake.valid?(:az322) && @az322_contribution.valid?
+        if came_from_old_page || (current_intake.valid?(:az322_form_create) && @az322_contribution.valid?)
           @az322_contribution.save
           redirect_to action: :index, return_to_review: params[:return_to_review]
         else
           render :new
         end
       end
 
+      # remove this after it has been deployed
```

Comment:
Can you make a ticket to remove this and add it to this sprint?

Reply:
[done](https://codeforamerica.atlassian.net/browse/FYST-1767?atlOrigin=eyJpIjoiOTBjZDZiNDNlOTlmNDM2NmExMzI4YmJiNTA0YWViOTEiLCJwIjoiaiJ9)!